main.js

```diff
@@ -0,0 +1,150 @@
+//操作するHTMLを取得
+const titleLogo = document.getElementById('titleLogo');
+const text = document.getElementById('text');
+const btn = document.getElementById('btn');
+const Api = 'https://opentdb.com/api.php?amount=10';
+//問題番号を管理する変数
+let quizNumber = 0;
+//正解数をカウントする変数
+let count = 0;
+//回答欄の親要素を作成
+const container = document.createElement('div');
+
+//ボタンを押したときにAPIを取得する
+btn.addEventListener('click', function () {
```

Comment:
全体的に言えるのですが、アロー関数を用いた方が最近の書き方です。

```suggestion
btn.addEventListener('click', () => {
```

Reply:
ありがとうございます。今後はアロー関数で書きます。